_pytest/doctest.py

```diff
@@ -103,8 +105,21 @@ def setup(self):
 
     def runtest(self):
         _check_all_skipped(self.dtest)
+        self._disable_output_capturing()
         self.runner.run(self.dtest)
 
+    def _disable_output_capturing(self):
```

Comment:
please include the platform name in the method name since its a platform hack
i propose `_disable_output_capturing_for_darwin`